etcdutl/go.mod

```diff
@@ -72,22 +72,22 @@ require (
 	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
 	go.etcd.io/etcd/client/v2 v2.306.0-alpha.0 // indirect
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.56.0 // indirect
-	go.opentelemetry.io/otel v1.31.0 // indirect
+	go.opentelemetry.io/otel v1.32.0 // indirect
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.31.0 // indirect
```

Comment:
Hey @ArkaSaha30 thanks for taking care of dependencies this week.  We have `go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.31.0` in the bumped list but it still shows as `v1.31.0`.  Can you please double check?

Reply:
I think the same applies to `go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc` (#18867)

Reply:
I actually have a slight doubt here, `otlptrace` and `otlptracegrpc` are purely indirect dependency but its parent package `go.opentelemetry.io/otel` is a direct dependency... so does it make sense to bump all the child components as well?

Reply:
I would lean towards bumping all of otel at the same time. This is a deviation from our purely indirect rule so we might want to update our dependency management guide to be clear on this.

cc @ahrtr , @ivanvc for opinions on this.

Reply:
> I would lean towards bumping all of otel at the same time. This is a deviation from our purely indirect rule so we might want to update our dependency management guide to be clear on this.

+1 to avoid any inconsistency even it's between parent and child .

Reply:
Sure, thank you. I will update and keep it consistent.